lib/alerts/unsubmitted_course_alert_manager.rb

```diff
@@ -8,26 +8,17 @@ def create_alerts
 
       alert = Alert.create(type: 'UnsubmittedCourseAlert',
                            course: course,
-                           user: course.instructors.first,
-                           target_user: alert_recipient(course))
-      alert.email_target_user
+                           user: course.instructors.first)
+      alert.send_email
     end
   end
 
   private
 
-  DAYS_AGO_LIMIT = 30
+  TIME_WINDOW = 3
   def unsubmitted_recently_started_courses
-    ClassroomProgramCourse.unsubmitted.where(start: DAYS_AGO_LIMIT.days.ago..Time.zone.today)
-  end
-
-  # First time instructors work with the Outreach Manager.
-  # Returning instructors work with the Classroom Program Manager.
-  def alert_recipient(course)
-    if course.tag? 'first_time_instructor'
-      SpecialUsers.outreach_manager
-    else # 'returning_instructor', and fallback just in case a course has neither tag.
-      SpecialUsers.classroom_program_manager
-    end
+    ClassroomProgramCourse.unsubmitted
+                          .where('courses.created_at <= ?', TIME_WINDOW.days.ago)
+                          .where('courses.start <= ?', TIME_WINDOW.days.from_now)
```

Comment:
It looks like this won't exclude old unsubmitted courses. We only want to do this basically at the point when a course is just nearing its start date. Maybe, following the pattern from the old version, `.where(start: TIME_WINDOW.days.ago..TIME_WINDOW.days.from_now)`.

Also, maybe either rename the constant or include the days in it (`TIME_WINDOW = 3.days`), just so that it's clear what exactly the value means.